### PR TITLE
Add formula for physfs 1.x.

### PR DESCRIPTION
**Pull request by wizonesolutions**
Needed to build ManaPlus on OS X.

### CODE REVIEW

**Comment by wizonesolutions** [comment]
Hmm; it installed locally. How can I test/fix these failures?

**Comment by DomT4** [comment]
```
==> audit problems
homebrew/versions/physfs1:
 * Description is too long. "name: desc" should be less than 80 characters.
Length is calculated as homebrew/versions/physfs1 + desc. (currently 81)
 * Dependency readline does not define option :test
 * Dependency readline does not define option :test
```

File: physfs1.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  homepage "https://icculus.org/physfs/"
+  url "https://icculus.org/physfs/downloads/physfs-1.1.1.tar.gz"
+  sha256 "994b65066657d49002509c9d84a9b0d2f5c0c1dab0457e9fe21ffa40f2205f63"
+
+  conflicts_with "physfs", :because => "Differing versions of the same formula"
+
+  head "https://hg.icculus.org/icculus/physfs/", :using => :hg
+
+  depends_on "cmake" => :build
+  depends_on "readline" => :test
```
Comment (DomT4) [inline physfs1.rb]: You can't have `test` only dependencies, essentially.

File: physfs1.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+class Physfs1 < Formula
+  desc "Library to provide abstract access to various archives"
+  homepage "https://icculus.org/physfs/"
+  url "https://icculus.org/physfs/downloads/physfs-1.1.1.tar.gz"
+  sha256 "994b65066657d49002509c9d84a9b0d2f5c0c1dab0457e9fe21ffa40f2205f63"
+
+  conflicts_with "physfs", :because => "Differing versions of the same formula"
+
+  head "https://hg.icculus.org/icculus/physfs/", :using => :hg
```
Comment (DomT4) [inline physfs1.rb]: Versions formulae don't have `head`s unless it points specifically to the branch of the version it is targeting.

**Pull request closed without merging**